### ISSUE TITLE
Very dark background for command blocks in Unity Pro Skin

### ISSUE THREAD

**Issue by DesperateHouseDev** [snozbot/fungus#813]
Hello, 

I'm facing an issue quite similar as issue #801 , except this is about commands background color: When editing a command block using the default Unity Pro skin (dark theme), every command has a very dark background color, where colors are almost indistinguishable from one another , while, when using Unity Personal skin (light theme), they have their expected colors.
See the attached screenshot.

**To Reproduce**
- Create a fresh Unity 2019.3 pro project (can't tell if this happens with earlier/later versions, but if there is a later version that corrects this issue, I'd be happy to know)
- Keep the default unity dark theme, or revert to it using Edit/preferences/general/Editor theme: professional
- Install latest (3.12.0) version of fungus
- Create a flowchart or open an example
- Create some commands in a Block
- See that these commands appear with a very dark background color in the editor, that makes every command type indistinguishable from another

**Expected behavior**
- The command background colors should be as clear as in unity personal (light theme)

**Screenshots**
![image](https://user-images.githubusercontent.com/51003327/78293304-af651380-7528-11ea-8503-2aea6ad15720.png)
![Command Background color issue](https://user-images.githubusercontent.com/51003327/78293719-58137300-7529-11ea-9f9a-69723133f23d.png)

**Versions & Platform **
 - Unity version 2019.3.0f3
 - Fungus Version v.3.12.0
 - Editor OS: Win 10
 - Build Platform Windows Standalone, Mono scripting backend

Thanks in advance for your help !

**Comment by stevehalliwell** [comment]
Thanks for the report. Unfortunately I don't have access to pro skin right now so I'm not able to confirm my suspicion nor that the fix is working. The command color tinting is done via the GUI.backgroundColor which is multiplied through. The previous color issue was prior to 2019.3, this issue appears to be new to 2019.3 and its new GUI styling.

My suspicion being that adding 
```GUI.color = Color.white;``` to CommandListAdapter.cs on line 333 may do the trick.

Are you able to determine cause/ if that change is having any effect?

**Comment by DesperateHouseDev** [comment]
Hello, 

Thanks a lot for your answer, and my apologies for not answering sooner myself.

No it seems that adding GUI.color = Color.white; to line 333 doesn't solve this issue; changing the color value here only slightly tints the background color of the command (e.g. very dark red if you choose Color.red, very dark green if you chose Color.green, etc...), as if any GUI color / Background color was "multiplied" by dark grey:
![image](https://user-images.githubusercontent.com/51003327/79074586-90673e00-7ced-11ea-995c-c2b508061e99.png)

**BUT :** After a few trials/error, I found out that the only effective way to set background color (and not only multiply it by their default color) on GUI Commands (GUI.Box(), GUI.Label(), GUI.Button(), etc...), was to **change their GUI.Style parameter**.

Your GUI.Label() commands (after line 333) are already using a GUI.Style named commandLabelStyle, which is set up line 27 to 40, that already correctly uses a background texture from the FungusEditorResources scriptable object, **but** this commandLabelStyle is initialized as `new GUIStyle(**GUI.skin.box**)` line 27, while you mainly use this Style to draw GUI Labels, not GUI Boxes. This _should_ work (to my poor knowledge), but, for some reason doesn't correctly lets you change the background texture as it should (at least not in Unity 2019.3...)
So I found out that changing line 27 from `commandLabelStyle = new GUIStyle(GUI.skin.box);` to `commandLabelStyle = new GUIStyle(GUI.skin.**label**);` does the trick : 

![image](https://user-images.githubusercontent.com/51003327/79075159-526c1900-7cf1-11ea-90e1-a0910a4aee1f.png)

As you can see, white commands titles are then barely readable as pointed in #801, but your fix to change `commandLabelStyle.normal.textColor` to `Color.black;` works like a charm.

![image](https://user-images.githubusercontent.com/51003327/79075271-256c3600-7cf2-11ea-8514-fde31a4d2d7a.png)


**IN SUMMARY** :   
Changing line 27 from : 
`commandLabelStyle = new GUIStyle(GUI.skin.box);`
to :
```
commandLabelStyle = new GUIStyle(GUI.skin.label);
commandLabelStyle.normal.textColor = Color.black;
```
seems like a perfectly acceptable Fix for this issue, and I recommend it.

Thanks again for your help !

**Comment by stevehalliwell** [comment]
Thanks very much for hunting that down in pro skin. I'll make that change in develop now.
